codingGuideLines/cpp.md

```diff
@@ -259,7 +259,6 @@ method(
     - parameter is surrounded by spaces ( regex: `functionName([:space:]param[:space:])` )
```

Comment:
we wanted to remove the "regex" note as well: written wording + example is enough
(also, the regex would be more complicated for proper newline + intend; it's incomplete)